lib/puppet-lint/report/codeclimate.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'digest'
+require 'json'
+
+class PuppetLint::Report
+  class CodeClimateReporter
+    def self.write_report_file(problems, report_file)
+      report = []
+      problems.each do |messages|
+        messages.each do |message|
+          case message[:kind].downcase
```

Comment:
The `downcase` is odd, given we're checking symbols. Is that really needed?

Reply:
Maybe I should slip in an extra commit to remove the wonkiness in that code.